react/src/lib/components/DeckGLMap/layers/axes2d/axes2DLayer.stories.tsx

```diff
@@ -8,7 +8,7 @@ export default {
 } as ComponentMeta<typeof DeckGLMap>;
 
 const layerProps = {
-    marginH: 80, // Horizontal margin (in pixels)
+    marginH: 100, // Horizontal margin (in pixels)
```

Comment:
I think we should revert it here or is there a specific reason we use 100 ?

Reply:
Horisontal margin needs to be a bit less than the vertical one so I think these are sensible default values,